lib/tasks/dev/prime.rake

```diff
@@ -1,7 +1,19 @@
 namespace :dev do
   desc "Sample data for local development environment"
   task prime: :environment do
-    campaign = Campaign.find_or_create_by(name: "Dragon of Icespire Peak")
+    dm = User.find_or_initialize_by(
+      email: "jdbann@icloud.com",
```

Comment:
@jdbann let me know if this isn't the right email or if you'd prefer not to have it here at all and we can find another way to do this...

Reply:
I don't mind, but maybe something more generic like `dm@table.com` would be easier for other contributors to remember?